### ISSUE TITLE
JAVA_ARGS ignored

### ISSUE THREAD

**Issue by Houpert** [voyages-sncf-technologies/tock#350]
Hello,

I'm testing actually the last tock docker (2.0.0) and it's looks like JAVA_ARGS is ignored for the docker `tock200_build_worker_1`.

Here is the configuration of the docker-compose (default has clone) :
      `- JAVA_ARGS=-Xmx1g`

But it's looks like the JVM use more than the configuration. It should restart if the memory goes above the 1G memory.
![memorytockbuild](https://user-images.githubusercontent.com/6594652/48339448-8ceb0580-e668-11e8-8059-6282309317d5.png)

Here is the trace of docker stats from the specific docker after running for the weekend
```
CONTAINER ID        NAME                        CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O           PIDS
e31e40a16c0c        tock200_build_worker_1      3.27%               1.77GiB / 7.615GiB    23.24%           1.48GB / 690MB      7.26MB / 3.32GB     58    
```

Thanks for the consideration!

**Comment by vsct-jburet** [comment]
@Houpert It seems -Xmx is not enough with java 8+. We need to set also -XX:MaxMetaspaceSize. With this commit https://github.com/voyages-sncf-technologies/tock-docker/commit/27922174022d3aebef5039070400e550fe97e6ce, I see consistent OutOfMemory errors: 

> 2018-11-12T20:47:37.966 [vert.x-worker-thread-1] ERROR fr.vsct.tock.nlp.front.service.ModelUpdaterService - Java heap space
java.lang.OutOfMemoryError: Java heap space

Let me know if it's also ok on your side.

**Comment by Houpert** [comment]
Hello

It seems that added the `-XX:MaxMetaspaceSize` has lock the memory to 1.247G.

The issue looks resolve for me.
I will continue to check if this move but i think this issue can be close atm.

Thanks.

**Comment by Houpert** [comment]
Hello,

After the server running the whole weekend, it's seem the memory leak is still here but slower this time.
The docker `tock200_build_worker_1` after running 6 days (date of the fix), the memory usage is at 1.9 Go

Docker stats :
![memoryfix](https://user-images.githubusercontent.com/6594652/48696641-c1b80900-ebe2-11e8-95de-9de82890113a.png)

and the htop (with oom data)
![htopleack](https://user-images.githubusercontent.com/6594652/48697072-ff696180-ebe3-11e8-8c92-66ad86e370c4.png)

and to finish the ps aux from the docker (Java_args looks fine)
![psauxdocker](https://user-images.githubusercontent.com/6594652/48697081-04c6ac00-ebe4-11e8-9695-8a06edf2118d.png)

Here some data of the VM if it's can help :

- uname -a : `Linux mvp-linstt 4.9.0-6-amd64 #1 SMP Debian 4.9.82-1+deb9u3 (2018-03-02) x86_64 GNU/Linux
- docker -v : `Docker version 18.03.1-ce, build 9ee9f40`
- docker-compose -v : `docker-compose version 1.21.2, build a133471`


I think i will try to build tock to see if this come from something else.


Thanks for the consideration !
